package.json

```diff
@@ -7,14 +7,15 @@
     "access": "public"
   },
   "scripts": {
+    "analyze": "wca analyze components/**/*.js --format json --outFile custom-elements.json",
```

Comment:
You may want to include `templates` as well to pick up our page templates... there's just one right now and it's not really finished though.